client/asset/dcr/dcr.go

```diff
@@ -1441,6 +1429,67 @@ func (dcr *ExchangeWallet) feeRateWithFallback(feeSuggestion uint64) uint64 {
 	return cfg.fallbackFeeRate
 }
 
+// FundsMixingStats returns the current state of the wallet's funds mixer. Part
+// of the asset.FundsMixer interface.
+func (dcr *ExchangeWallet) FundsMixingStats(ctx context.Context) (*asset.FundsMixingStats, error) {
+	var canControlMixer, isMixing bool
+	if mixer, ok := dcr.wallet.(fundsMixer); ok {
+		canControlMixer = true
+		isMixing = mixer.IsMixing()
+	}
+
+	accts := dcr.wallet.Accounts()
+	stats := &asset.FundsMixingStats{
+		CanControlMixer:         canControlMixer,
+		Enabled:                 accts.UnmixedAccount != "",
+		Active:                  isMixing,
+		UnmixedBalanceThreshold: smalletCSPPSplitPoint,
+	}
+
+	var minConf int32
+	primaryAcctBalance, err := dcr.wallet.AccountBalance(ctx, 0, accts.PrimaryAccount)
+	if err != nil {
+		return nil, err
+	}
+
+	if !stats.Enabled {
+		// Wallet not configured for mixing (yet). Return the default account
+		// balance only.
+		stats.UnmixedBalance = toAtoms(primaryAcctBalance.Total) // assume default balance as unmixed
+		return stats, nil
+	}
+
+	minConf = 1 // dcrwallet MixAccount uses minConf=1 to select mixable inputs
+	unmixedBalance, err := dcr.wallet.AccountBalance(ctx, minConf, accts.UnmixedAccount)
```

Comment:
Right. It would. This isn't used for normal balance reporting though. But we can still use minConf=0, after all those funds are unmixed and would be mixed when they get the required confs. Will update.